src/main/kotlin/com/sourcegraph/cody/chat/UIComponents.kt

```diff
@@ -17,4 +18,11 @@ object UIComponents {
     button.setUI(buttonUI)
     return button
   }
+
+  @JvmStatic
+  fun createMainButton(text: String, icon: Icon): JButton {
```

Comment:
if the only usage of `createMainButton` is this case, and if we were able to bring both the text and the icon into the enum, this method could take only the one enum argument - I do not see the wider scope here, but pls take it into consideration

Reply:
Im not sure about including an image into an enum, seems more button related to me 😅